src/main/java/woowa/sw/baseball/UserImpl.java

```diff
@@ -0,0 +1,33 @@
+package woowa.sw.baseball;
+
+import java.util.Scanner;
+
+public class UserImpl implements User{
+    private static final int USER_INPUT_CNT = 3;
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    @Override
+    public String[] getUserRandomBall() throws IllegalArgumentException {
+        System.out.print("숫자를 입력하세요 : ");
+        String[] arrUserInput = SCANNER.nextLine().replaceAll(" ", "").split("");
```

Comment:
```suggestion
        String[] arrUserInput = SCANNER.nextLine()
            .replaceAll(" ", "")
            .split("");
```

`.` 단위로 개행을 해주면 좀 더 가독성 있는 코드가 됩니다.